packages/framework/src/Model/Customer/CustomerFacade.php

```diff
@@ -66,7 +72,8 @@ public function __construct(
         CustomerMailFacade $customerMailFacade,
         BillingAddressFactoryInterface $billingAddressFactory,
         DeliveryAddressFactoryInterface $deliveryAddressFactory,
-        BillingAddressDataFactoryInterface $billingAddressDataFactory
+        BillingAddressDataFactoryInterface $billingAddressDataFactory,
+        UserFactoryInterface $userFactory
```

Comment:
I am thinking, is not this also BC break that should be mentioned in `UPGRADE.md`? It is true, that autowiring takes care for that but imagine the scenario when somebody already extended `CustomerFacade` in their project and overridden the constructor - it might cause him problems.

Reply:
ok.... so do You suggest adding all classes that changed constructor into upgrade notes? Well.... I don't want to do that, because there would be huge number of classes that are maybe a little relevant for someone.... Hm... how strongly do you think this is important?